src/Akeneo/Search/UpdatedCriteria.cs

```diff
@@ -12,7 +12,7 @@ public static UpdatedCriteria Equals(DateTime date)
 			return new UpdatedCriteria
 			{
 				Operator = Operators.Equal,
-				Value = date
+				Value = date.ToString("yyyy-MM-dd HH:mm:ss")
```

Comment:
I'm a bit hesitant to add a call to `ToString` here. Is this date format accepted throughout the Akeneo API? If so, we should update the [`AkeneoSerializerSettings`](https://github.com/pardahlman/akeneo-csharp/blob/master/src/Akeneo/Serialization/AkeneoSerializerSettings.cs) to set [`DateFormatHandling`](http://www.newtonsoft.com/json/help/html/DatesInJSON.htm) accordingly.